runtime/hydradx/src/system.rs

```diff
@@ -246,14 +247,14 @@ impl pallet_collator_selection::Config for Runtime {
 	type UpdateOrigin = MoreThanHalfCouncil;
 	type PotId = PotId;
 	type MaxCandidates = MaxCandidates;
-	type MinCandidates = MinCandidates;
 	type MaxInvulnerables = MaxInvulnerables;
 	// should be a multiple of session or things will get inconsistent
 	type KickThreshold = Period;
 	type ValidatorId = <Self as frame_system::Config>::AccountId;
 	type ValidatorIdOf = pallet_collator_selection::IdentityCollator;
 	type ValidatorRegistration = Session;
 	type WeightInfo = weights::collator_selection::HydraWeight<Runtime>;
+	type MinEligibleCollators = ConstU32<4>;
```

Comment:
where does the 4 come from ? 

cc @mrq1911 as he knows stuff about collators better

Reply:
from the parachain-template

Reply:
could be set even higher, but that could cause issues with testnets. if that is an cases we will probably set it to 1.

@Roznovjak did you check if zombienet and chopsticks produce blocks regularly after session change?

Reply:
Didn't notice any issue using chopsticks